modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-type-select/src/main/resources/META-INF/resources/select_field.js

```diff
@@ -185,6 +187,12 @@ AUI.add(
 
 						var instance = this;
 
+						var container = instance.get('container');
+
+						var selectGroup = container.one('.form-builder-select-field');
+
+						selectGroup.addClass('active');
+
 						if (!instance.get('readOnly')) {
 							instance.get('container').one('.drop-chosen').toggleClass('hide');
```

Comment:
You guess is necessary? It is just a behavior to show a feedback to the user, showing a blue bottom border when this input has focus.

I'll send another commit with the suggestions you've said.